src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3549,9 +3549,14 @@ void MusicXMLParserDirection::pedal(const QString& type, const int /* number */,
             p->setLineVisible(false);
         }
         if (!p->lineVisible() || sign == "yes") {
-            p->setBeginText(u"<sym>keyboardPedalPed</sym>");
-            p->setContinueText(u"(<sym>keyboardPedalPed</sym>)");
+            p->setBeginText(Pedal::PEDAL_SYMBOL);
+            p->setContinueText(Pedal::STAR_SYMBOL);
```

Comment:
STAR_SYMBOL as the continue text?

Reply:
My bad, fixed.

Reply:
Caused an mtest to fail in my backport, in "testPedalChangesBroken", I wonder why it didn't here?